lib/scaffold.dart

```diff
@@ -211,6 +211,16 @@ class BackdropScaffold extends StatefulWidget {
   /// Will be called when [backLayer] has been revealed.
   final VoidCallback? onBackLayerRevealed;
 
+  /// Specifies whether the state of the [backLayer] is maintained when it is
+  /// revealed and concealed.
+  ///
+  /// When true (default), the [backLayer] is kept in the tree while concealed.
+  /// When false, the [backLayer] is removed from the tree when concealed and
+  /// recreated when revealed.
+  ///
+  /// Defaults to `true`.
+  final bool maintainState;
```

Comment:
should we rename this to make it more particular - `maintainBackLayerState` ?

Reply:
Done